### ISSUE TITLE
About GOPRO dataset

### ISSUE THREAD

**Issue by DachunKai** [XiangZ-0/EVDI#12]
In the READEME.md, 'GoPro contains synthetic blurry images and synthetic events. We first convert [REDS](https://seungjunnah.github.io/Datasets/reds.html) into high frame rate videos.' **Why use Gopro, but use Reds to generate videos?Is it wrong?**

**Comment by XiangZ-0** [comment]
Hi, thanks for the question. 
This is correct. We name the dataset following the previous work [eSL-Net](https://arxiv.org/pdf/2007.08336.pdf), where the event-based dataset built upon REDS is also called GoPro since the REDS dataset is collected using GoPro cameras. 
Hope this helps.

**Comment by DachunKai** [comment]
@XiangZ-0 Thanks, I thought your GoPro dataset was the one commonly used in video deblurring, following this [link](https://seungjunnah.github.io/Datasets/gopro.html). It seems it's not the same?

**Comment by XiangZ-0** [comment]
Yes, our dataset is built on [the 2019 dataset REDS](https://seungjunnah.github.io/Datasets/reds) instead of [the 2017 one](https://seungjunnah.github.io/Datasets/gopro.html). Sorry about the confusion caused. We use a new name in our recent deblurring work [GEM](https://github.com/XiangZ-0/GEM) to avoid confusion. Welcome to check them 😄
